package/MDAnalysis/analysis/hole.py

```diff
@@ -1276,29 +1306,17 @@ def _process_orderparameters(self, data):
             raise ValueError(errmsg)
         return q
 
-    def run(self, start=None, stop=None, step=None):
```

Comment:
This looks as if it reverted an existing PR in which start/stop/step were made compliant.

Reply:
Probably wants a `git rebase origin/master`.  start/stop/step are now only in `run()` and never `__init__`, so `self.start/step/stop` can never exist

Reply:
Yes, forgot to rebase.